app/models/maestrano/connector/rails/organization.rb

```diff
@@ -74,5 +74,9 @@ def last_three_synchronizations_failed?
     def last_successful_synchronization
       self.synchronizations.where(status: 'SUCCESS', partial: false).order(updated_at: :desc).first
     end
+
+    def last_synchronization_date
```

Comment:
Should this actually be
```ruby
(last_successful_synchronization && last_successful_synchronization.updated_at) || date_filtering_limit
```
so the last successful synchronisation takes precedence over the date filtering limit?